test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -37,43 +39,90 @@ public void IrisLightGbm()
                 new DatabaseLoader.Column() { Name = "PetalWidth", Type = DbType.Single }
             };
 
-            using (var connection = new MockConnection(mlContext, GetDataPath(TestDatasets.iris.trainFilename), loaderColumns))
+            var loader = mlContext.Data.CreateDatabaseLoader(loaderColumns);
+
+            var mockProviderFactory = new MockProviderFactory(mlContext, loaderColumns);
+            var databaseSource = new DatabaseSource(mockProviderFactory, connectionString, commandText);
+
+            var trainingData = loader.Load(databaseSource);
+            //trainingData = mlContext.Data.Cache(trainingData, "Label", "SepalLength", "SepalWidth", "PetalLength", "PetalWidth");
```

Comment:
I think we can remove these commented out lines.

Reply:
Fixed.